helpers/image_manipulation/training_sample.py

```diff
@@ -691,7 +691,7 @@ def get_conditioning_image(self):
         conditioning_dataset = StateTracker.get_conditioning_dataset(
             data_backend_id=self.data_backend_id
         )
-        print(f"Conditioning dataset components: {conditioning_dataset.keys()}")
+        # print(f"Conditioning dataset components: {conditioning_dataset.keys()}")
```

Comment:
Maybe we can just log this information formally?

Reply:
it wasn't needed, but it's nice to have sometimes commented out like this if i'm adding a new model or training / validation style